user/deviceid/deviceid.go

```diff
@@ -0,0 +1,17 @@
+package deviceid
+
+import (
+	"encoding/base64"
+
+	"github.com/getlantern/golog"
+	"github.com/google/uuid"
+)
+
+var (
+	log = golog.LoggerFor("desktop.deviceid")
+)
+
+// OldStyleDeviceID returns the old style of device ID, which is derived from the MAC address.
+func OldStyleDeviceID() string {
```

Comment:
I don't think we need to worry about this anymore -- won't be any old style IDs kicking around this code base